modules/integrate-with-clair.adoc

```diff
@@ -5,17 +5,16 @@
 [id="integrate-with-clair_{context}"]
 = Integrating with Clair
 
-You can integrate {product-title} with Clair for the static analysis of vulnerabilities in your images.
+Beginning with release 4.4, Clair scanning features are available in the new {product-title-short} scanner, Scanner V4, and do not require a separate integration. The instructions in this section are only required if you are using the Clair V4 scanner.
 
 [NOTE]
 ====
-* With {product-title-short} 3.74, you can integrate with Clair V4 Scanner.
-Red Hat has deprecated the previous CoreOS Clair integration in favor of Clair v4 integration.
+* With {product-title-short} 3.74, a separate integration was required to use the Clair V4 Scanner. This integration is no longer required if you are using Scanner V4 with {product-title-short} release 4.4 and later.
```

Comment:
We shouold probably still keep the note that CoreOS Clair integration is still deprecated